spec/lib/docker-sync/dependencies/package_managers/none_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+RSpec.describe DockerSync::Dependencies::PackageManager::None do
+  it_behaves_like 'a dependency'
+
+  describe '.available?' do
+    subject { described_class.available? }
+    it { is_expected.to be true }
+  end
+
+  describe '.ensure!' do
+    subject { described_class.ensure! }
+    it { is_expected.to execute_nothing }
+  end
+
+  describe '.install_package(package_name)' do
+    let(:user_confirmed?) { true }
+    let(:package_name)    { 'some-package' }
+
+    before do
+      allow(described_class).to receive(:ensure!)
+      allow_any_instance_of(Thor::Shell::Color).to receive_messages(
+        say_status: nil,
+        yes?: user_confirmed?
+      )
+    end
+
+    subject { described_class.install_package(package_name) }
+
+    it 'asks for user confirmation' do
```

Comment:
You're very right, ignoring errors is a bad habbit of mine 😞 
Let me fix this 👍